products/big-ip.md

```diff
@@ -0,0 +1,44 @@
+---
+title: BIG-IP 
+category: os
+permalink: /big-ip
+versionCommand: show /sys version
+releaseDateColumn: true
+eolColumn: End of Technical Support
+eoasColumn: End of Software Development
+
+releases:
+
+-   releaseCycle: "17.1.x"
```

Comment:
```suggestion
-   releaseCycle: "17.1"
```

We usually don't add `.x` on release names, could you update all `releaseCycle` fields accordingly ?

Reply:
fixed